modules/apps/frontend-theme/frontend-theme-cms/src/css/components/_sticker.scss

```diff
@@ -0,0 +1,15 @@
+.cms-sticker-dropdown {
+	&:focus,
+	&:hover {
+		background-color: $gray-100;
+		border-color: $gray-600;
+		font-size: 0;
+
+		&:after {
+			background-image: url(@theme_image_path@/clay/caret-double.svg);
```

Comment:
We don't want to use `@theme_image_path@` anymore. Although this works with a theme, it won't work when it gets converted to a themeCSS CX since Liferay can't replace a token like this when it's a CX. Instead we can just use a relative url to theme images folder.

Reply:
Changed!

<img width="433" alt="image" src="https://github.com/user-attachments/assets/19ecd1cd-1c61-4bd2-a2c1-df1a2487414b" />